### PR TITLE
Meenu/ eutestlink trading specification

### PR DESCRIPTION
**Pull request by meenakshi-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/trading-specification/components/_elements.tsx [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+}
+
+export const StyledButton = styled(Button)`
+    border: none;
+    background: transparent;
+`
+export const StyledButtonPage = styled(Button)<{ selected: boolean }>`
+    border-radius: 4px;
+    background: transparent;
+    color: #414652;
```
Comment (NikitK-deriv) [inline src/pages/trading-specification/components/_elements.tsx:255]: Should we use non-standard color?

File: src/pages/trading-specification/components/_popup_menu.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            }
+        })
+    }, [market])
+
+    return (
+        <>
+            <Card>
+                {is_calculated ? (
+                    <ModalCard>
+                        <Header type="subtitle-1" weight="700" as="p">
```
Comment (NikitK-deriv) [inline src/pages/trading-specification/components/_popup_menu.tsx:53]: `weight=700` by default in Header

File: src/pages/trading-specification/components/_trading_spec_table.tsx [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+        width: 95%;
+    }
+`
+const SearchForm = styled.form`
+    position: relative;
+    align-items: center;
+    padding: 6px 8px;
+    width: 464px;
+    height: 32px;
+    border: 1px solid #d6dadb;
```
Comment (NikitK-deriv) [inline src/pages/trading-specification/components/_trading_spec_table.tsx:52]: please check it as well, we have to use only standard colors. if in design different, we have to ask designers about it or change standard colors

File: src/pages/trading-specification/components/_trading_spec_table.tsx [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+        getCoreRowModel: getCoreRowModel(),
+        getSortedRowModel: getSortedRowModel(),
+        onSortingChange: setSorting,
+        getPaginationRowModel: getPaginationRowModel(),
+        onGlobalFilterChange: setGlobalFilter,
+    })
+
+    const handleSubmit = (e) => {
+        e.preventDefault()
+    }
```
Comment (NikitK-deriv) [inline src/pages/trading-specification/components/_trading_spec_table.tsx:129]: Shouldn't we set smth?

File: src/pages/trading-specification/index.tsx [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+import Layout from 'components/layout/layout'
+import { SectionContainer, SEO } from 'components/containers'
+import { Header } from 'components/elements'
+import MarketTab from 'components/elements/market-tab'
+import { Localize, localize, WithIntl } from 'components/localization'
+import device from 'themes/device'
+
+const StyledHeader = styled(Header)`
+    text-align: center;
+    font-size: 48px;
```
Comment (NikitK-deriv) [inline src/pages/trading-specification/index.tsx:12]: it's exactly `type='heading-2'`

File: src/pages/trading-specification/index.tsx [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    }
+`
+const TradingSpecification = () => {
+    return (
+        <Layout>
+            <SEO
+                title={localize('Trading specifications | CFD trading | Trading with Deriv')}
+                description={localize(
+                    'Find Deriv’s CFD trading specifications for multiple assets in forex, derived indices, stocks and indices, cryptocurrencies, and commodities.',
+                )}
```
Comment (NikitK-deriv) [inline src/pages/trading-specification/index.tsx:26]: Exclusively for SEO component we can write `localize` function
```suggestion
                title={localize('_t_Trading specifications | CFD trading | Trading with Deriv_t_')}
                description={localize(
                    '_t_Find Deriv’s CFD trading specifications for multiple assets in forex, derived indices, stocks and indices, cryptocurrencies, and commodities._t_',
                )}
```

Comment (meenakshi-deriv) [reply src/pages/trading-specification/index.tsx]: Hi @NikitK-deriv, fixed all these issues in new commit. Thanks :)

**Pull request closed without merging**